static_files/index.html

```diff
@@ -16,37 +16,33 @@
     <!-- Header -->
 
     <header>
-      <div>
-
-        <!-- Logo -->
-
-        <div>
+<!-- Always Visible Header -->
+      <div class="HeaderContainer">
+        <div class="CathoLogo">
           <a href="#"
             ><img
               src="../static_files/imgs/cathoLogo.svg"
               alt="Logotipo da Catho"
           /></a>
         </div>
-        <!-- Main Navigation Menu -->
-        <nav>
+        <nav class="VisibleMenu">
           <ul>
             <li><a href="#">Catho Components</a></li>
             <li><a href="#">Quantum</a></li>
+            <li>&#x2630;</li>
           </ul>
         </nav>
+      </div>
 
-        <!-- Hidden Navigation Menu -->
+<!-- Hidden Navbar -->
+
+      <nav class="HiddenMenu">
+        <ul>
+          <li><a href="#">Contact us</a></li>
+          <li><a href="#">Repositories</a></li>
+        </ul>
+      </nav>
```

Comment:
How about aligning the list of items relative to the element nav? I keep in mind that this way the implementation of the resources will be faithfully aligned to the prototype and with that we avoid divergent spacing between these items and the visible menu above.
Implementation:
![image](https://user-images.githubusercontent.com/105747728/234895430-2c5f8dfc-e888-42ed-9bac-fca3ec3d08b9.png)
Prototype
![image](https://user-images.githubusercontent.com/105747728/234895704-0c769d73-4f50-41cd-8174-424637af54f3.png)

Reply:
Fixed